javascript/example/DRACOLoader.js

```diff
@@ -15,6 +15,7 @@
 'use strict';
 
 THREE.DRACOLoader = function(manager) {
+    this.decoder = THREE.DRACOLoader.getDecoder();
```

Comment:
is it necessary to store the decoder as a member variable? Isn't `THREE.DRACOLoader.getDecoder()` sufficient? (i.e. replacing instances of `this.decoder;` with `this.getDecoder();`)